frontend/src/components/Header.tsx

```
@@ -0,0 +1,68 @@
+import { alpha, AppBar, Box, Container, IconButton, InputBase, styled, Toolbar, Typography } from '@mui/material';
+import SearchIcon from '@mui/icons-material/Search';
+import ClearIcon from '@mui/icons-material/Clear';
+//import React from 'react';
+
+type Props = {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)